docs/src/webknossos-py/examples/apply_merger_mode.md

```diff
@@ -0,0 +1,13 @@
+# Apply Merger Mode Annotation
+
+This example applies a merger mode annotation to a segmentation.
+
+It downloads a [sample dataset](https://static.webknossos.org/data/l4_segmentation.zip) and uses the [Dataset API](dataset_usage.md) for reading and rewriting the `segmentation` layer.
```

Comment:
Please explain a bit more about merger mode, that it generates the nml and link to the docs:
https://docs.webknossos.org/webknossos/volume_annotation.html#proof-reading-and-merging-segments

Is this a useful example which actually solves some splits? Then we could also link to the original dataset and annotation and the result, and maybe add a screenshot.